src/kdbindings/signal.h

```diff
@@ -401,8 +305,8 @@ class Signal
      */
     void disconnect(const ConnectionHandle &handle)
     {
-        if (m_impl && handle.belongsTo(*this)) {
-            m_impl->disconnect(handle.m_id);
+        if (m_impl && handle.belongsTo(*this), handle.m_id.has_value()) {
```

Comment:
Hm, why is this using `,` instead of `&&`?
Doesn't that skip the result of the first statement?

That's one of those weird C-syntax edge cases that I'm not terribly familiar with, so I might be wrong here :)